guides/data/nav/users.yml

```diff
@@ -21,12 +21,12 @@
     dropdown:
       - title: "Overview"
         href: "/users/promotions/overview.html"
+      - title: "Promotion Rules"
+        href: "/users/promotions/promotion-rules.html"
       - title: "Promotion Actions"
         href: "/users/promotions/promotion-actions.html"
       - title: "Promotion Calculators"
         href: "/users/promotions/promotion-calculators.html"
-      - title: "Promotion Rules"
-        href: "/users/promotions/promotion-rules.html"
```

Comment:
Is this change really necessary for this PR? I'm assuming most users would expect to have an alphabetically-sorted menu, what do you think?

Reply:
I would expect administrators to read about "Promotion Rules" before they read about "Promotion Actions." Perhaps my commit message should have stated that more explicitly.

While the guides may not be read linearly, we can prioritize articles by putting them up higher in the menus. We do this with "Overview" articles.

This is a low-risk change so I will include it. We can revert later if we formalize menu orders.

Reply:
I think it makes sense to read rules before actions since this is also the order we are presenting information in the UI.